server/controllers/auth.js

```diff
@@ -2,8 +2,8 @@ const User = require("../models/User");
 const Profile = require("../models/Profile");
 const asyncHandler = require("express-async-handler");
 const generateToken = require("../utils/generateToken");
-const dotenv = require('dotenv').config({path:__dirname+'/./../.env'})
-const loginHelper = require('../helpers/loginHelper');
+const dotenv = require("dotenv").config({ path: __dirname + "/./../.env" });
```

Comment:
We wouldn't need to use this here, once the env is attached to the process from the `bin/www` startup script it will be available anywhere else in the codebase. 

If you could just remove it - I know you weren't the one who added it but that would be appreciated 💯